tests/integration/run.sh

```diff
@@ -18,7 +24,13 @@ ${ROOT_DIR}/occ app:enable spreed || exit 1
 ${ROOT_DIR}/occ app:enable spreedcheats || exit 1
 ${ROOT_DIR}/occ app:list | grep spreed
 
+# Disable bruteforce protection because the integration tests do trigger them
+${ROOT_DIR}/occ config:system:set auth.bruteforce.protection.enabled --value false --type bool
```

Comment:
Is this needed in Talk tests too?

Reply:
As soon there will be bruteforce protection being added to the endpoint, yes.